web/client/components/background/BackgroundSelector.jsx

```diff
@@ -1,290 +1,284 @@
 /*
- * Copyright 2017, GeoSolutions Sas.
+showLabel * Copyright 2017, GeoSolutions Sas.
  * All rights reserved.
  *
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
 
```

Comment:
There are `.css` files inside the `web/client/components/background/`, please remove them if they are not used anymore